headers/types/dungeon_mode/dungeon.h

```diff
@@ -672,8 +674,17 @@ struct dungeon {
     bool standing_in_kecleon_shop;
     undefined field_0x7ca;
     undefined field_0x7cb;
-    bool boost_max_money_amount; // 0x7CC: Boost the floor's maximum Poké limit by 8x
-    uint8_t _padding_0x7cd[3];
+    // 0x7CC: Controls which mappa file (Time/Darkness/Sky) to load for a dungeon. When time and
+    // darkness are selected their respective mappa files are loaded, the hidden stairs type
+    // will always return 0, treasure boxes will generate with only exclusive items that exist
+    // in time/darkness, use a multiple of 0x28 (40) for poke. If sky is selected, causes
+    // waza_p2.bin to be loaded over waza_p.bin. Because waza_p.bin is initally loaded into
+    // the move table and the check to load waza_p2.bin is later, waza_p2.bin can be deleted
+    // without causing the game to crash as the data from waza_p.bin is still loaded because
+    // it's not overwritten by loading waza_p2.bin
+    // 0 = Sky, 1 = Time, 2 = Darkness
```

Comment:
Can we make an `enum` for this instead of using `uint32_t`? Something like

```c
enum game {
    GAME_SKY = 0,
    GAME_TIME = 1,
    GAME_DARKNESS = 2,
};
```

(We can always rename this later if we need to)